bccsp/schemes/idemixopts.go

```diff
@@ -249,9 +249,24 @@ type NymEIDAuditData struct {
 	EID *math.Zr
 }
 
+// RHAuditData contains the data that is used to audit the RH.
+// Notice that this data should be used only after validating the corresponding signature.
+type RHAuditData struct {
+	// CommRH is the commitment to randomness
```

Comment:
this comment doesn't seem correct - this is the commitment to the revocation handle right?

Reply:
I will open a new PR since this one is quite outdated and there have been numerous changes